app.js

```diff
@@ -176,7 +176,7 @@ exports.init = function() {
     app.use('/health', healthcheck);
 
     if (useIDAM === 'true') {
-        app.use(/\/((?!error)(?!sign-in)(?!pin-resend)(?!pin-sent)(?!co-applicant-*)(?!pin)(?!inviteIdList).)*/, security.protect(config.services.idam.roles));
+        app.use(/\/((?!error)(?!sign-in)(?!pin-resend)(?!pin-sent)(?!co-applicant-*)(?!pin)(?!inviteIdList)(?!start-page).)*/, security.protect(config.services.idam.roles));
```

Comment:
I found a way we can move this list of pages to an array in the config and then use them here. I'll run through this tomorrow.